src/main/java/controller/LoginController.java

```diff
@@ -22,7 +22,13 @@ void doPost(HttpRequest request, HttpResponse response) throws Exception {
                 Cookie.SET_COOKIE,
                 "logined" + KEY_VALUE_DELIMITER + String.valueOf(isLogin) + COOKIE_DELIMITER + COOKIE_PATH + KEY_VALUE_DELIMITER + "/"
         );
-        response.sendRedirect(isLogin ? "/index.html" : "/login_failed.html");
+
+        if (isLogin) {
+            response.sendRedirect("/index.html");
+        }
+        if (!isLogin) {
+            response.sendRedirect("/login_failed.html");
+        }
```

Comment:
```suggestion
        if (!isLogin) {
            response.sendRedirect("/login_failed.html");
            return;
        }
        
        response.sendRedirect("/index.html");
```
이렇게도 코드를 만들 수 있을 것 같아요. :)

Reply:
그렇네요 ㅎㅎ if 문을 두번이나 쓸 이유가 없었네요